packages/perspective-viewer-d3fc/src/js/interaction/clickHandler.js

```diff
@@ -0,0 +1,56 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2017, the Perspective Authors.
+ *
+ * This file is part of the Perspective library, distributed under the terms of
+ * the Apache License 2.0.  The full license can be found in the LICENSE file.
+ *
+ */
+
+import {interpolate} from "d3";
```

Comment:
Click handling is another thing that seems to be related to the series (see other comments).